flow/raster_cache.cc

```diff
@@ -34,8 +34,19 @@ void RasterCacheResult::draw(SkCanvas& canvas, const SkPaint* paint) const {
 
   SkRect bounds =
       RasterCacheUtil::GetDeviceBounds(logical_rect_, canvas.getTotalMatrix());
-  FML_DCHECK(std::abs(bounds.width() - image_->dimensions().width()) <= 1 &&
-             std::abs(bounds.height() - image_->dimensions().height()) <= 1);
+#ifndef NDEBUG
+  // The image dimensions should always be larger than the device bounds and
+  // smaller than the device bounds plus one pixel, at the same time, we must
+  // introduce epsilon to solve the round-off error. The value of epsilon is
+  // 1/512, which represents half of an AA sample.
+  //
+  // see https://github.com/flutter/flutter/issues/107622
```

Comment:
please inline all the useful information rather than linking to the bug database. We don't know if the bug database will outlive the source code.